app/controllers/users_controller.rb

```diff
@@ -1651,4 +1654,8 @@ def confirm_secure_session
   def secure_session_confirmed?
     secure_session["confirmed-password-#{current_user.id}"] == "true"
   end
+
+  def summary_cache_key(user)
+    "user_summary:#{user.id}:#{current_user ? current_user.id : 0}"
```

Comment:
```suggestion
    "user_summary:#{user.id}:#{current_user ? current_user.id : 0}"
```
Trivial but `to_i` would work the same way here. `nil.to_i` is 0.